lib/ezutils/classes/ezmodule.php

```diff
@@ -857,8 +857,9 @@ function redirectionURIForModule( $module, $viewName, $parameters = array(),
             $uri .= $unorderedURI;
         }
 
-        if ( is_array( $userParameters ) )
+        if ( is_array( $userParameters ) && count( $userParameters ) > 0 )
```

Comment:
nitpick: you should also be able to use `!empty()`, and put it before `is_array`. it's language construct, so vastly faster.